.github/workflows/tests.yml

```diff
@@ -7,9 +7,9 @@ on:
     - cron: "0 0 * * *"
 
 jobs:
-  build:
+  tests:
```

Comment:
The job also builds some configurations without running any tests (i.e the non-emulated platforms). It only runs a few tests. So this job's name can remain `build` or be changed to `build_and_test`, to be more correct.